### PR TITLE
:sparkles: Source Zendesk Support: add stream Ticket Skips

### PR DESCRIPTION
**Pull request by davydov-d**
## What
https://github.com/airbytehq/airbyte/issues/16888
Add `Ticket skips` stream

## How
- Add new stream
- Add it to existing unit tests
- Add it to existing CAT
- Bump connector version

### CODE REVIEW

File: airbyte-integrations/connectors/source-zendesk-support/acceptance-test-config.yml [hunk 1]
```diff
@@ -31,6 +31,8 @@ acceptance_tests:
           # This stream is only available for enterprise accounts https://developer.zendesk.com/api-reference/ticketing/account-configuration/audit_logs/
           - name: "audit_logs"
             bypass_reason: "no records"
+          - name: "ticket_skips"
```
[the remark this answers is not in the document]
Comment (davydov-d) [reply airbyte-integrations/connectors/source-zendesk-support/acceptance-test-config.yml:34]: @igrankova can you help?

**Comment by octavia-squidington-iii** [comment]
##  source-zendesk-support test report (commit [`0a7918a548`](https://github.com/airbytehq/airbyte/pull/27436/commits/0a7918a54821d92c063eb5c99999b94b16e7a883)) - ✅

⏲️  Total pipeline duration: 1383 seconds

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zendesk-support/metadata.yaml | ✅        |
| Connector version semver check.                                               | ✅        |
| Connector version increment check.                                            | ✅        |
| QA checks                                                                     | ✅        |
| Connector package install                                                     | ✅        |
| Build source-zendesk-support docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ✅        |
| Acceptance tests                                                              | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5291544005)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

**Comment by octavia-squidington-iii** [comment]
##  source-zendesk-support test report (commit [`0a7918a548`](https://github.com/airbytehq/airbyte/pull/27436/commits/0a7918a54821d92c063eb5c99999b94b16e7a883)) - ✅

⏲️  Total pipeline duration: 1078 seconds

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zendesk-support/metadata.yaml | ✅        |
| Connector version semver check.                                               | ✅        |
| Connector version increment check.                                            | ✅        |
| QA checks                                                                     | ✅        |
| Connector package install                                                     | ✅        |
| Build source-zendesk-support docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ✅        |
| Acceptance tests                                                              | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5291544821)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

**Comment by octavia-squidington-iii** [comment]
##  source-zendesk-support test report (commit [`e7c1e3b271`](https://github.com/airbytehq/airbyte/pull/27436/commits/e7c1e3b2715a98bee61e16d5439e963f8073920e)) - ❌

⏲️  Total pipeline duration: 39 seconds

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zendesk-support/metadata.yaml | ✅        |
| Connector version semver check.                                               | ✅        |
| Connector version increment check.                                            | ✅        |
| QA checks                                                                     | ✅        |
| Connector package install                                                     | ✅        |
| Build source-zendesk-support docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ❌        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5292290083)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

**Comment by octavia-squidington-iii** [comment]
##  source-zendesk-support test report (commit [`a3e785f2be`](https://github.com/airbytehq/airbyte/pull/27436/commits/a3e785f2bee81eb905e734317dc61db02418cf9d)) - ❌

⏲️  Total pipeline duration: 937 seconds

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zendesk-support/metadata.yaml | ✅        |
| Connector version semver check.                                               | ✅        |
| Connector version increment check.                                            | ✅        |
| QA checks                                                                     | ✅        |
| Connector package install                                                     | ✅        |
| Build source-zendesk-support docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ✅        |
| Acceptance tests                                                              | ❌        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5292706330)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

File: airbyte-integrations/connectors/source-zendesk-support/acceptance-test-config.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (davydov-d) [reply airbyte-integrations/connectors/source-zendesk-support/acceptance-test-config.yml:34]: @lazebnyi this is a very tricky stream, our unblockers team did their best but could not seed it

**Comment by octavia-squidington-iii** [comment]
##  source-zendesk-support test report (commit [`d0477fe9f7`](https://github.com/airbytehq/airbyte/pull/27436/commits/d0477fe9f7e288a73f254580f9d6146125efd515)) - ❌

⏲️  Total pipeline duration: 1039 seconds

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zendesk-support/metadata.yaml | ✅        |
| Connector version semver check                                                | ✅        |
| Connector version increment check                                             | ✅        |
| QA checks                                                                     | ✅        |
| Code format checks                                                            | ❌        |
| Connector package install                                                     | ✅        |
| Build source-zendesk-support docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ✅        |
| Acceptance tests                                                              | ❌        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/ddavydov_16888-source-zendesk-support-add-stream-ticket-skips/1687853012/d0477fe9f7e288a73f254580f9d6146125efd515/source-zendesk-support/0.6.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

File: airbyte-integrations/connectors/source-zendesk-support/acceptance-test-config.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (davydov-d) [reply airbyte-integrations/connectors/source-zendesk-support/acceptance-test-config.yml]: @lazebnyi the stream has been seeded

**Comment by octavia-squidington-iii** [comment]
##  source-zendesk-support test report (commit [`1102713fb7`](https://github.com/airbytehq/airbyte/pull/27436/commits/1102713fb73f7a44ee1bdab55e7153c36781bb3b)) - ❌

⏲️  Total pipeline duration: 1158 seconds

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zendesk-support/metadata.yaml | ✅        |
| Connector version semver check                                                | ✅        |
| Connector version increment check                                             | ✅        |
| QA checks                                                                     | ✅        |
| Code format checks                                                            | ✅        |
| Connector package install                                                     | ✅        |
| Build source-zendesk-support docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ✅        |
| Acceptance tests                                                              | ❌        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/ddavydov_16888-source-zendesk-support-add-stream-ticket-skips/1687856493/1102713fb73f7a44ee1bdab55e7153c36781bb3b/source-zendesk-support/0.6.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

**Review by lazebnyi — APPROVED** [review]

**Comment by octavia-squidington-iii** [comment]
##  source-zendesk-support test report (commit [`92d1f732f0`](https://github.com/airbytehq/airbyte/pull/27436/commits/92d1f732f014337ddd8bd256ab5dc3749e3a221a)) - ❌

⏲️  Total pipeline duration: 923 seconds

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zendesk-support/metadata.yaml | ✅        |
| Connector version semver check                                                | ✅        |
| Connector version increment check                                             | ✅        |
| QA checks                                                                     | ✅        |
| Code format checks                                                            | ✅        |
| Connector package install                                                     | ✅        |
| Build source-zendesk-support docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ✅        |
| Acceptance tests                                                              | ❌        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/ddavydov_16888-source-zendesk-support-add-stream-ticket-skips/1687859852/92d1f732f014337ddd8bd256ab5dc3749e3a221a/source-zendesk-support/0.6.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

**Comment by octavia-squidington-iii** [comment]
##  source-zendesk-support test report (commit [`beea0b268a`](https://github.com/airbytehq/airbyte/pull/27436/commits/beea0b268a70f80789aea81ae85725333243b128)) - ✅

⏲️  Total pipeline duration: 1659 seconds

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zendesk-support/metadata.yaml | ✅        |
| Connector version semver check                                                | ✅        |
| Connector version increment check                                             | ✅        |
| QA checks                                                                     | ✅        |
| Code format checks                                                            | ✅        |
| Connector package install                                                     | ✅        |
| Build source-zendesk-support docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ✅        |
| Acceptance tests                                                              | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/ddavydov_16888-source-zendesk-support-add-stream-ticket-skips/1687863852/beea0b268a70f80789aea81ae85725333243b128/source-zendesk-support/0.6.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

**Comment by octavia-squidington-iii** [comment]
##  source-zendesk-support test report (commit [`3cd618fd46`](https://github.com/airbytehq/airbyte/pull/27436/commits/3cd618fd462f25e8d9d57bb95f6166960cfd16c0)) - ✅

⏲️  Total pipeline duration: 951 seconds

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zendesk-support/metadata.yaml | ✅        |
| Connector version semver check                                                | ✅        |
| Connector version increment check                                             | ✅        |
| QA checks                                                                     | ✅        |
| Code format checks                                                            | ✅        |
| Connector package install                                                     | ✅        |
| Build source-zendesk-support docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ✅        |
| Acceptance tests                                                              | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/ddavydov_16888-source-zendesk-support-add-stream-ticket-skips/1687867087/3cd618fd462f25e8d9d57bb95f6166960cfd16c0/source-zendesk-support/0.7.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

**Pull request merged**